tests/clipper_test.go

```diff
@@ -0,0 +1,276 @@
+package clipper
+
+import (
+	"os"
+	"testing"
+
+	"github.com/atotto/clipboard"
+	"github.com/supitsdu/clipper/cli/clipper"
+
+	"github.com/xyproto/randomstring"
+)
+
+const mockTextContent = "Mocking Bird! Just A Sample Text."
+
+func TestContentReaders(t *testing.T) {
+	t.Run("FileContentReader", func(t *testing.T) {
+		// Create a temporary file
+		file, err := createTempFile(t, mockTextContent)
+		if err != nil {
+			t.Fatalf("Failed to create temp file: %v", err)
+		}
+
+		// Create a FileContentReader
+		reader := clipper.FileContentReader{FilePath: file.Name()}
+
+		// Read the content
+		readContent, err := reader.Read()
+		if err != nil {
+			t.Fatalf("Error reading file: %v", err)
+		}
+
+		// Check the content
+		if readContent != mockTextContent {
+			t.Errorf("Expected '%s', got '%s'", mockTextContent, readContent)
+		}
+	})
+
+	t.Run("StdinContentReader", func(t *testing.T) {
+		// Create a StdinContentReader
+		reader := clipper.StdinContentReader{}
+
+		// Replace stdin with a pipe
+		_, w := replaceStdin(t)
+
+		// Write some content to the pipe
+		_, err := w.WriteString(mockTextContent)
+		if err != nil {
+			t.Fatalf("Error writing to pipe: %v", err)
+		}
+
+		// Close the write end of the pipe
+		err = w.Close()
+		if err != nil {
+			t.Fatalf("Error closing pipe: %v", err)
+		}
+
+		// Read the content
+		readContent, err := reader.Read()
+		if err != nil {
+			t.Fatalf("Error reading from stdin: %v", err)
+		}
+
+		// Check the content
+		if readContent != mockTextContent {
+			t.Errorf("Expected '%s', got '%s'", mockTextContent, readContent)
+		}
+	})
+}
+
+func TestClipboardWriter(t *testing.T) {
+	t.Run("DefaultClipboardWriter", func(t *testing.T) {
+		if testing.Short() == true {

```

Comment:
```suggestion
		if testing.Short() {
```

Reply:
Also, I wouldn't have uses short for this, but an environment variable

https://stackoverflow.com/questions/24030059/skip-some-tests-with-go-test

https://dev.to/salesforceeng/subtesting-skipping-and-cleanup-in-the-go-testing-t-49ea